src/utils.coffee

```diff
@@ -44,13 +44,18 @@ class Utils
     result = [result] unless result instanceof Array
     _.compact(result)
 
-  @wrapObjects: (array) ->
+  @wrapObjects: (array, options = {}) ->
     output = []
     _(array).each (elem) =>
       if elem.constructor == Object
         for key, value of elem
           o = {}
-          o[key] = @wrapObjects(if value instanceof Array then value else [value])
+
+          if options.ignoreCollections && elem[key] instanceof Backbone.Collection
```

Comment:
Here is an example of how `wrapObjects` used to work:
```
include: [
                  'assignees',
                  { project: projectCollection },
                  { sub_tasks: ["assignees"] }
                ]
```
converts to:
![screen shot 2017-05-08 at 12 01 28 pm](https://cloud.githubusercontent.com/assets/2433091/25820447/1a6516bc-33e6-11e7-8d85-eba9ef4a6541.png)